ckanext/data_qld/dataset_deletion/helpers.py

```diff
@@ -0,0 +1,40 @@
+import ckan.lib.helpers as h
```

Comment:
@awset Remember to use the toolkit for helpers etc instead of importing them directly.
Check the toolkit file before importing any ckan core files